session_2/student_registry/app/models/student.rb

```diff
@@ -0,0 +1,9 @@
+class Student < ApplicationRecord
+  validates_presence_of :name, :roll_number, :branch, :address, :admission_year, :cgpa, message: 'cannot be null',
+                                                                                        on: :create
```

Comment:
The `on: :create` is incorrect - Can a student have a nil roll number after creation?

Reply:
so, it's better to leave it on default? that is, on save?

Reply:
Yeah